data/fixtures/portals/github/datapackage.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
I think we don't need to put the test packates under `fixtures/portals/github` we can just have `data/package-name` and use them across the integrations (ckan/github/zenodo/etc)

Reply:
I would also have at leaset 2 resources there

Reply:
I removed those files and used the existing files datapackage.json.

It has two resources in it.